HISTORY.md

```diff
@@ -1,171 +1,171 @@
-2.0.0-beta.1 / 2020-03-29
+`2.0.0-beta.1` / 2020-03-29
```

Comment:
These headers shouldn't have the backticks; it deviates from all the other history file formats and breaks our tooling.

Reply:
Sorry - couldn't resist the temptation. Reverted.